testing/jormungandr-integration-tests/src/common/jcli/api/key.rs

```diff
@@ -81,7 +81,7 @@ impl Key {
             .as_single_line()
     }
 
-    pub fn to_public_expect_fail<S: Into<String>>(self, private_key: S, expected_msg_path: &str) {
+    pub fn into_public_expect_fail<S: Into<String>>(self, private_key: S, expected_msg_path: &str) {
```

Comment:
This does not look like an `into_*`, either. Maybe `convert_to_public_expect_fail`?